crates/driver/src/infra/config/file/mod.rs

```diff
@@ -97,7 +97,7 @@ fn default_additional_tip_percentage() -> f64 {
 }
 
 fn default_gas_price_cap() -> f64 {
-    1500.0
+    1e6
 }
```

Comment:
1500 is very low and doesn't get settled on hardhat or geth. Changed it to 1 gwei.